modules/es-extensions/publisher/extensions/app/greg_impact/public/js/canvg.js

```diff
@@ -73,7 +73,9 @@
 
         svg.log = function(msg) {};
         if (svg.opts['log'] == true && typeof(console) != 'undefined') {
-            svg.log = function(msg) { console.log(msg); };
```

Comment:
We do not need to comment this out as the library takes care of handling console object not been present